client/src/main/java/io/pravega/client/tables/impl/TableSegmentKey.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.tables.impl;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Key for a {@link TableSegment}.
+ */
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.PACKAGE)
+public class TableSegmentKey {
```

Comment:
Same question about the visibility of this class.

Reply:
Same answer.